base/qserv.yaml

```diff
@@ -5,11 +5,13 @@ metadata:
 spec:
   storageclass: "standard"
   storagecapacity: "1Gi"
+  czar:
+    image: "qserv/qserv:dcbfff7"
```

Comment:
are we ever going to be able to get away from using these hex-tagged image ids and move toward some kind of meaningful label, e.g. `"qserv/qserv:latest"`? (Does that make sense or am I thinking about it wrong?)

Reply:
Rolling tag like "latest" are difficult to manage at large scale because you will not know what run on a given platform, and it is difficult to retrieve the source code related to "latest" tag. However I think "latest" exists and is available for development purpose. So fixed that looks better to me, current hex-tag is related to a git hash, so that it is easy to retrieve the corresponding source code. The regular way is to manage release tag with version number, maybe we should define a process for creating release tag?